demos/elastic-newstracker/README.md

```diff
@@ -6,112 +6,109 @@ This demo loads a data set of Internet news snippets from April 2009, collected
-the elastic container over the named link `eshost`. The juttle programs will be accessible inside the container at `/opt/juttles` mount point.
-
-To verify that the outrigger container can in fact reach the elastic search container, run this juttle query via CLI inside `outrigger`:
+the ``docker-compose.yml`` contains everything necessary to run all
+the containers for this demo. Assuming you are in the directory where
+this README is located, simply run:
 
 ```
-docker exec outrigger juttle -e "read elastic -from :2009-04-01: -to :2009-05-01: | reduce count()"
+docker-compose up
 ```
 
-At this point, there is no data in elasticsearch, therefore you should see zero count of records.
-
-## Load Demo Data
-
-Unzip the memes_data.zip and then use logstash to load this data into your running elasticsearch container:
+If you're running docker via sudo, you also need to ensure that the
+environment variable PWD is the current directory:
 
 ```
-unzip memes_data.zip
-cat memes_data.json | docker run -i --rm -v `pwd`:/config --link elastic:eshost logstash:2.1.1 -f /config/logstash.conf -w 4
+sudo PWD=$PWD docker-compose up
 ```
 
-*Note:* the previous command will take several minutes to execute as we're
-loading quite a bit of data into elasticsearch. Proceed with the next steps while this is running.
+This will download and start the containers with appropriate links
+between them.
 
-## Verify Data
+This maps the following files to the following containers:
 
-While data is being ingested, re-running the Juttle query should show a rising count of records.
+Logstash:
+- ``logstash.conf`` -> ``/config/logstash.conf``
+- ``news_data.zip`` -> ``/incoming/news_data.zip``
 
-```
-docker exec outrigger juttle -e "read elastic -from :2009-04-01: -to :2009-05-01: | reduce count()"
-```
+Outrigger:
+- The juttle programs in juttles -> ``/opt/outrigger/juttles``
+- ``juttle-config.json`` -> ``/opt/outrigger/.juttle-config.json``
 
-When ingest has completed, you should see the output:
+## Juttles
 
-```
-┌───────────┐
-│ count     │
-├───────────┤
-│ 1200500   │
-└───────────┘
-```
+For easy navigation of the Juttle programs, visit
+``http://localhost:8080/run?path=/index.juttle``, which will output a
+table with links in your browser. Click the links to run the example
+programs.
 
+If you're running using docker-machine, replace localhost with the
+value of ``docker-machine ip default``.
 
-## Juttles
+### See the number of elasticsearch items
 
-For easy navigation of the Juttle programs, run this one, which will output a table with links in your browser. Click the links to run the demo programs. They will work assuming you added 'docker-local' entry to your `/etc/hosts`.
+To see the number of elasticsearch items parsed by logstash, run the
+program ``num_elastic_points.juttle`` by clicking on the
+``num_elastic_points`` link from the table. You should see a non-zero
+increasing count of records while logstash populates elasticsearch
```

Comment:
it's not increasing by itself. Let's add `(re-run the program with Run button, or your browser refresh button, to see updated results)`.